src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -3992,17 +3992,6 @@ CorInfoIntrinsics ZapInfo::getIntrinsicID(CORINFO_METHOD_HANDLE method,
                                           bool * pMustExpand)
 {
     CorInfoIntrinsics intrinsicID = m_pEEJitInfo->getIntrinsicID(method, pMustExpand);
-
-#if defined(TARGET_X86) || defined(TARGET_AMD64)
```

Comment:
Is there a replacement of this code?

Reply:
We are always returning `CORINFO_INTRINSIC_Illegal` ID for named math intrinsic.

Reply:
But the JIT will recognize this intrinsic by name and will generate the hardware-specific code for it.

@davidwrighton Do you have a suggestion for the best way to do the same when these intrinsics are recognized by name?

Reply:
@am11 I believe that this logic needs to move to FilterNamedIntrinsicMethodAttribs in this file.

Reply:
I've updated zapinfo.cpp, incorporating both feedback comments.